lms/djangoapps/course_home_api/dates/v1/serializers.py

```diff
@@ -39,6 +39,6 @@ class DatesTabSerializer(serializers.Serializer):
     """
     course_date_blocks = DateSummarySerializer(many=True)
     display_reset_dates_text = serializers.BooleanField()
-    learner_is_verified = serializers.BooleanField()
+    learner_is_full_access = serializers.BooleanField()
```

Comment:
This is an API break, heads up. Only affects our not-yet-public MFE dates tab, but still. Not saying don't make the change, but maybe you should also update the MFE before finishing the ticket.

Reply:
Oh wait, nope. The MFE doesn't use that field yet (it only uses the per-item `learner_has_access` fields). Never mind. Change away!